### PR TITLE
CI - NERD Workflow Modifications + GRANTS GPMATS Automation

### PR DESCRIPTION
**Pull request by iamdez99**

### CODE REVIEW

File: src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RAS_Screener_Submissions_StepsImpl.java
Comment (Mariachaudhry) [inline src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RAS_Screener_Submissions_StepsImpl.java:1]: Line 23 needs to be removed

File: src/test/java/ServiceNow/CHARMS/StepsImplementation/TestAccountResetImpl.java
Comment (Mariachaudhry) [inline src/test/java/ServiceNow/CHARMS/StepsImplementation/TestAccountResetImpl.java:1]: line 34 needs to be removed

**Review by Mariachaudhry — CHANGES_REQUESTED** [review]

File: src/test/java/GrantsApps/GPMATS/Steps/GPMATS_All_Steps.java [hunk 1]
```diff
@@ -0,0 +1,1058 @@
+package GrantsApps.GPMATS.Steps;
```
Comment (hamidnesar) [inline src/test/java/GrantsApps/GPMATS/Steps/GPMATS_All_Steps.java:1]: There are a lot of empty lines throughout this page within the code starting from line 58, 61, 64, .... Are those empty lines there on purpose to enhance readability?

File: src/test/java/GrantsApps/GrantsRunners/GPMATS/Run_GPMATS_Progression_Test.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package GrantsApps.GrantsRunners.GPMATS;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+
+@CucumberOptions(plugin = {"html:target/GPMATS-progression-reports/GPMATS-progression-report.html", "json:target/cucumber.json",
+        "junit:target/cucumber.xml", "rerun:target/failed.txt",
```
Comment (hamidnesar) [inline src/test/java/GrantsApps/GrantsRunners/GPMATS/Run_GPMATS_Progression_Test.java:7]: Please remove "junit:target/cucumber.xml"

File: src/test/java/GrantsApps/GrantsRunners/GPMATS/Run_GPMATS_Regression_Test.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package GrantsApps.GrantsRunners.GPMATS;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+
+@CucumberOptions(plugin = {"html:target/GPMATS-regression-reports/GPMATS-regression-report.html", "json:target/cucumber.json",
+            "junit:target/cucumber.xml", "rerun:target/failed.txt",
```
Comment (hamidnesar) [inline src/test/java/GrantsApps/GrantsRunners/GPMATS/Run_GPMATS_Regression_Test.java:7]: please remove "junit:target/cucumber.xml"

File: .gitattributes [hunk 4]
```diff
@@ -0,0 +1 @@
+*.html filter=lfs diff=lfs merge=lfs -text
```
Comment (hamidnesar) [inline .gitattributes:1]: Is this a new class introduced in the framework? Wondering if it is meant to be added to git ignore

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi, 
Please review the comments. Thanks!

**Review by Mariachaudhry — APPROVED** [review]

**Review by hamidnesar — APPROVED** [review]
Reviewed and approved

**Review by alenaminsk — APPROVED** [review]

**Pull request merged**